### ISSUE TITLE
Tweaks to scr-top dist tarball

### ISSUE THREAD

**Issue by adammoody** [LLNL/scr-top#9]
Given a "dist" tarball from scr-top, our current build instructions look like:
```
mkdir scr
cd scr
tar -xzf ../scr-top-v3.0rc1.tgz
make unpack
make build
cd build
cmake -DCMAKE_BUILD_TYPE=Release ../
make -j install
```
For more conventional usage, it'd be helpful to change things so that one could instead do:
```
tar -xzf scr-top-v3.0rc1.tgz
cd scr-top-v3.0rc1
mkdir build install
cd build
cmake -DCMAKE_BUILD_TYPE=Release ../
make -j install
```
Have the tarball create the ``scr-top-v3.0rc1`` directory when it extracts.  Drop the internal source tarballs so a single tar command extracts all files to eliminate ``make unpack`` step.  Have user ``mkdir`` the build and install directories to replace the ``make build`` step.

**Comment by gonsie** [comment]
question about the dist tarball: do you want the repos inside it to include the git files or not?

**Comment by adammoody** [comment]
I ended up including a change for this issue with the PR you just merged, so we can close this now.

My goal for the dist tarball that we hand to users and the one I've attached to the SCR release page is that it's free of any git files -- it only has source files.  This is the case for ``make dist`` and ``make pack-lite``, where ``dist`` still gives someone the most recent tagged version of each package and ``pack-lite`` gives the latest source code for each package.

The ``make pack`` still currently zips up all git files.
